onnxmltools/convert/coreml/shape_calculators/Classifier.py

```diff
@@ -43,7 +43,7 @@ def calculate_traditional_classifier_output_shapes(operator):
     if operator.targeted_onnx_version < StrictVersion('1.2'):
         output_shape = [1, 1]
     else:
-        output_shape = [N, 1]
+        output_shape = [N]
```

Comment:
I believe this new bahavior needs to be clarified in ONNX spec as well. Basically, the current spec says nothing about this. If it doesn't match previous ONNX behavior, version number should be increased too.

Reply:
One reason to have 2-D is to make sure regressor and classifier have the same shape interface. For multi-output regression, its output shape must be [N, C]. If we consider classifier is a special case of regression, its output shape may be [N, 1] here.